package.json

```diff
@@ -90,6 +91,7 @@
     "pino-pretty": "4.7.1",
     "sanitize-html": "2.3.2",
     "set-interval-async": "1.0.34",
+    "smoothscroll-polyfill": "0.4.4",
```

Comment:
You need to install [here](https://github.com/Seneca-CDOT/telescope/blob/master/src/web/package.json)
Not in the root `package.json`

Reply:
Right, I just realized